utils.py

```diff
@@ -165,3 +164,19 @@ def getCountDBObjects(dbHost, dbName, login, password):
 	result["params"] = params[0][0]
 	result["locals"] = locals[0][0]
 	return result
+
+
+def getAmounts(dbHost, dbName, login, password):
```

Comment:
Amounts of what? getUserTokenAmounts?

Reply:
second, wouldn't it be better not to select just amounts, but SUM(amounts)?

Reply:
First - fixed, second - no, it's not better, because we should to compare records by users, not by sum.